src/Bicep.Core.UnitTests/Registry/DescriptorFactoryTests.cs

```diff
@@ -11,7 +11,7 @@
 namespace Bicep.Core.UnitTests.Registry
 {
     [TestClass]
-    public class DescriptorFactoryTests
+    public class OciDescriptorTests
```

Comment:
renamed this test class since I had previously erased the DescriptorFactory, moved its functions to a couple of static methods in OciDescriptor to reduce complexity